### PR TITLE
grace 5.1.25

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: grace.rb [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+  depends_on "fftw"
+  depends_on "lesstif"
 
   def install
     ENV.O1 # https://github.com/Homebrew/homebrew/issues/27840#issuecomment-38536704
     system "./configure", "--disable-debug",
                           "--prefix=#{prefix}",
                           "--enable-grace-home=#{prefix}"
+    system "make", "install"
+    share.install "fonts", "examples"
```
Comment (DomT4) [inline grace.rb]: We tend to make examples optional, but will allow it by default here since X11 stuff is a pain to test.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-x11/commit/065f714fdc00a7fd04cb2a4ea1df5703df8a57b2, Thanks!

**Comment by DomT4** [comment]
Side note: Please make sure you change the commit messages as well as the PR title if the commit is worded incorrectly. I totally missed this before merging, so "my bad" here.

**Comment by dunn** [comment]
Oh crap, sorry about that!

**Pull request closed without merging**